package.json

```diff
@@ -30,7 +30,7 @@
     "recompose": "^0.26.0"
   },
   "optionalDependencies": {
-    "lodash.orderby": "^4.6.0",
+    "lodash": "^4.17.0",
     "sortabular": "^1.5.1",
```

Comment:
Is there a reason to pull in all of lodash? Looks like only orderby is being used here.

Reply:
I'm using also `cloneDeep` and `findIndex`.

Reply:
OK, can we move this to devDependencies then?

Reply:
Sure, moved.